client/src/components/modals/createDataset/DataFileDataSourceSettings.jsx

```diff
@@ -17,6 +18,12 @@ export default class DataFileDataSourceSettings extends Component {
     this.handleDragEnter = this.handleDragEnter.bind(this);
     this.handleDragOver = this.handleDragOver.bind(this);
     this.handleDrop = this.handleDrop.bind(this);
+    this.handleProgress = this.handleProgress.bind(this);
+    this.state = { uploadProgressPercentage: null };
+  }
+
+  getShowProgressBarBool() {
```

Comment:
I would suggest renaming this method to remove the type suffix. Perhaps simply `showProgressBar`? or `isProgressBarVisible`? I don't think we do much data type suffix/prefix based naming currently and I like it that way.